### DISCUSSION TITLE
NUOVO AGGIORNAMENTO?

### DISCUSSION THREAD

**Discussion by scuolagioco** [iisgiua/giuaschool#244] [Idee e proposte]
Buon pomeriggio, quando sarà disponibile un nuovo aggiornamento per il registro?

**Comment by trinko** [comment]
Sto lavorando sulle modifiche allo scrutinio finale. Quando finisco le procedure carico l'aggiornamento.

**Comment by trinko** [comment]
No, l'esame di stato è escluso. Anche perché si usa l'applicativo ministeriale Commissione Web per gestirlo.
